drivers/sensor/bmi160/bmi160_spi.c

```diff
@@ -0,0 +1,71 @@
+/* Bosch BMI160 inertial measurement unit driver
+ *
+ * Copyright (c) 2020 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Datasheet:
+ * https://ae-bst.resource.bosch.com/media/_tech/media/datasheets/BST-BMI160-DS000.pdf
+ */
+
+#include <init.h>
+#include <drivers/sensor.h>
+#include <sys/byteorder.h>
+#include <kernel.h>
+#include <sys/__assert.h>
+#include <logging/log.h>
+
+#include "bmi160.h"
+
+#define BMI160_CHIP_ID		0xD1
+#define DT_DRV_COMPAT		bosch_bmi160
+
+LOG_MODULE_REGISTER(BMI160, CONFIG_SENSOR_LOG_LEVEL);
+
+struct bmi160_device_data bmi160_data = {
+	.chipid = BMI160_CHIP_ID,
+};
+
+static const struct sensor_driver_api bmi160_api = {
+	.attr_set = bmi160_attr_set,
+#ifdef CONFIG_BMI160_TRIGGER
+	.trigger_set = bmi160_trigger_set,
+#endif
+	.sample_fetch = bmi160_sample_fetch,
+	.channel_get = bmi160_channel_get,
+};
+
+int bmi160_init(struct device *dev)
+{
+	struct bmi160_device_data *bmi160 = dev->driver_data;
+	int ret;
+
+	bmi160->spi = device_get_binding(DT_INST_BUS_LABEL(0));
+	if (!bmi160->spi) {
+		LOG_DBG("SPI master controller not found: %s.",
+			    DT_INST_BUS_LABEL(0));
+		return -EINVAL;
+	}
+
+	bmi160->spi_cfg.operation = SPI_WORD_SET(8);
+	bmi160->spi_cfg.frequency = DT_INST_PROP(0, spi_max_frequency);
+	bmi160->spi_cfg.slave = DT_INST_REG_ADDR(0);
+
+	ret =  bm160_device_init(dev);
+	if (ret)
+		LOG_DBG("device init fail: %d", ret);
```

Comment:
`{}` should be used on all if/else. ref. https://docs.zephyrproject.org/latest/contribute/index.html#coding-style